src/lib/ReveryTerminal.rei

```diff
@@ -29,12 +29,16 @@ module Font: {
 module Screen: {
   type t;
 
+  let initial: t;
+
   let getCell: (~row: int, ~column: int, t) => Vterm.ScreenCell.t;
+  let getScreenCell: (~row: int, ~column: int, t) => Vterm.ScreenCell.t;
```

Comment:
This name confuses me. If this retrieves a a cell within the screen, does `getCell` retrieve a cell outside the screen? Why is it in the `Screen` module then? Should this perhaps be named `getVisibleCell` instead?

Reply:
Makes sense, I can see why it is confusing. I removed `getScreenCell` from the public API, and renamed it to `getVisibleRows` internally.